src/fast_check/transform.rs

```diff
@@ -1194,6 +1430,7 @@ impl<'a> FastCheckTransformer<'a> {
           None
         }
       }
+      Expr::Paren(n) => self.maybe_infer_type_from_expr(&n.expr),
```

Comment:
Is there a test for this?

Reply:
Yes - https://github.com/denoland/deno_graph/blob/e22155973b1d687fcdcb984154919e59873c1bb6/tests/specs/graph/JsrSpecifiers_FastCheck_Vars.txt#L56